src/app/proto/validator/accounts/v2/web_api.ts

```diff
@@ -209,9 +191,13 @@ export interface ImportKeystoresResponse {
   importedPublicKeys: string[];
 }
 
-export interface CreateAccountsRequest {
+export interface ValidateKeystoresRequest {
   /**
-   *  Number of accounts to create.
+   *  JSON encoded keystore list.
    */
-  numAccounts: number;
+  keystores: string[];
+  /**
+   *  Password to unlock the keystores.
+   */
```

Comment:
Is this a standard comment format? Seems a bit bulky.

Reply:
Yep standard, it is generated by proto-gen-ts